ssl/tls13_enc.c

```diff
@@ -499,10 +499,26 @@ int tls13_change_cipher_state(SSL_CONNECTION *s, int which)
             labellen = sizeof(client_early_traffic) - 1;
             log_label = CLIENT_EARLY_LABEL;
 
-            handlen = BIO_get_mem_data(s->s3.handshake_buffer, &hdata);
-            if (handlen <= 0) {
-                SSLfatal(s, SSL_AD_INTERNAL_ERROR, SSL_R_BAD_HANDSHAKE_LENGTH);
-                goto err;
+#ifndef OPENSSL_NO_ECH
+            /* if ECH worked then use the innerch and not the h/s buffer here */
+            if ((which & SSL3_CC_SERVER && s->ext.ech.success == 1)
```

Comment:
Same priority issue on these two lines

Reply:
adding parenthesis is fine and matches other code blocks so did that